packages/rum-core/src/domain/action/computeFrustration.ts

```diff
@@ -17,14 +17,17 @@ export function computeFrustration(clicks: Click[], rageClick: Click) {
   }
 
   const hasSelectionChanged = clicks.some((click) => click.getUserActivity().selection)
+  const didScrollOccur = clicks.some((click) => click.getUserActivity().scroll)
   clicks.forEach((click) => {
     if (click.hasError) {
       click.addFrustration(FrustrationType.ERROR_CLICK)
     }
     if (
       isDead(click) &&
       // Avoid considering clicks part of a double-click or triple-click selections as dead clicks
-      !hasSelectionChanged
+      !hasSelectionChanged &&
+      // Avoid considering clicks that resulted in a scroll as dead clicks
+      !didScrollOccur
```

Comment:
**❓ question:** ‏what's the reasoning behind this condition? I'm not sure to get the comment because `isDead` is already checking for `scroll` user activity.

Reply:
You're completely write, I missed that the first thing we check is if the click `isDead`, in which the check for `scroll` activity is done.